tasks/jasmine-chromeapp.js

```diff
@@ -7,73 +7,169 @@ module.exports = function (grunt) {
     chalk = require('chalk'),
     path = require('path'),
     async = require('async'),
-    fs = require('fs-extra'),
+    fs = require('fs'),
     pkg = require('../package.json'),
-    chrome = require('node-chrome-runner');
+    chrome = require('node-chrome-runner'),
+    fileSyncCmp = require('file-sync-cmp');
 
-  function addFiles(files, to, tagFilter) {
-    var tags = '';
+  /* copyFiles is adapted from:
```

Comment:
have a new file that does a `module.exports = function(grunt)` similar to how this one is structured?

or `require('grunt')` might work, since it looks like the dependence is mostly on static things like `.util`